k-distribution/tutorial/1_k/2_imp/lesson_5/imp.md

```diff
@@ -1,37 +1,37 @@
 // Copyright (c) 2014-2019 K Team. All Rights Reserved.
 
-/*!
-\setlength{\parindent}{1em}
-\title{IMP}
-\author{Grigore Ro\c{s}u (\texttt{grosu@illinois.edu})}
-\organization{University of Illinois at Urbana-Champaign}
-*/
-
-/*@ \section{Abstract}
-This is the \K semantic definition of the classic IMP language.
+IMP
+===
+
+Author: Grigore Roșu (grosu@illinois.edu)  
+Organization: University of Illinois at Urbana-Champaign
+
+### Abstract
+This is the **K** semantic definition of the classic IMP language.
 IMP is considered a folklore language, without an official inventor,
 and has been used in many textbooks and papers, often with slight
-syntactic variations and often without being called IMP\@.  It includes
+syntactic variations and often without being called **IMP**.  It includes
 the most basic imperative language constructs, namely basic constructs
 for arithmetic and Boolean expressions, and variable assignment,
 conditional, while loop and sequential composition constructs for statements.
-*/
 
+```k
 module IMP-SYNTAX
   imports DOMAINS-SYNTAX
-/*@ \section{Syntax}
-This module defines the syntax of IMP\@.
-Note that \texttt{<=} is sequentially strict and has a \LaTeX\ attribute
-making it display as $\leq$, and that \texttt{\&\&} is strict only in its first
-argument, because we want to give it a short-circuit semantics. */
+```
+### Syntax
+This module defines the syntax of **IMP**.
+Note that `<=` is sequentially strict, and `&&` is strict only in its first
+argument, because we want to give it a short-circuit semantics.
 
+```k
   syntax AExp  ::= Int | Id
                  | "-" Int                    [format(%1%2)]
                  | AExp "/" AExp              [left, strict, color(pink)]
                  > AExp "+" AExp              [left, strict, color(pink)]
                  | "(" AExp ")"               [bracket]
   syntax BExp  ::= Bool
-                 | AExp "<=" AExp             [seqstrict, latex({#1}\leq{#2}), color(pink)]
+                 | AExp "<=" AExp             [seqstrict, color(pink)]
```

Comment:
We should keep the `\latex...` output attribute, because that can still affect the `--output latex` option to KAST in the future (does not right now, but will).

Reply:
Note that this is a separate thing than the sources being in LaTeX.